include/freerdp/settings.h

```diff
@@ -1559,6 +1562,9 @@ struct rdp_settings
 	                                   default value - currently UNUSED! */
 	ALIGN64 char* ActionScript;
 	ALIGN64 DWORD Floatbar;
+
+	/* resource management related options */
+	ALIGN64 UINT32 ThreadingFlags;
```

Comment:
Better move to API stable zone with a specific position. Run the `update-*` scripts under `scripts` to generate getter/setter